Samples/Android/Speech Devices SDK Starter App/example/app/src/main/java/com/microsoft/coginitiveservices/speech/samples/sdsdkstarterapp/conversation.java

```diff
@@ -0,0 +1,388 @@
+package com.microsoft.coginitiveservices.speech.samples.sdsdkstarterapp;
+
+import android.content.Intent;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.text.Layout;
+import android.text.TextUtils;
+import android.text.method.ScrollingMovementMethod;
+import android.util.Log;
+import android.util.Pair;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.WindowManager;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import com.microsoft.cognitiveservices.speech.ResultReason;
+import com.microsoft.cognitiveservices.speech.SpeechConfig;
+import com.microsoft.cognitiveservices.speech.audio.AudioConfig;
+import com.microsoft.cognitiveservices.speech.conversation.ConversationTranscriber;
+import com.microsoft.cognitiveservices.speech.conversation.ConversationTranscriptionEventArgs;
+import com.microsoft.cognitiveservices.speech.conversation.Participant;
+import com.microsoft.cognitiveservices.speech.conversation.User;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.math.BigInteger;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Properties;
+import java.util.Random;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class conversation extends AppCompatActivity {
+    private HashMap<String, String> signatureMap = new HashMap<>();
+    private HashMap<String, Integer> colorMap = new HashMap<>();
+    private TextView IntermediateTextView;
+    private static final String CTSKey = "<Conversation Transcription Service Key>";
+    private SpeechConfig speechConfig = null;
+    private static final String inroomEndpoint = "<Conversation Transcription Service Endpoint>";
```

Comment:
inroomEndpoint is not needed anymore after you use fromSubscription

Reply:
I will update the modification on next release which is based on carbon V.1.51